backend/Terraform/modules/management_api/rest_api_getway.tf

```diff
@@ -0,0 +1,158 @@
+resource "aws_api_gateway_rest_api" "management_api" {
+  name        = var.api_name
+  endpoint_configuration {
+    types = ["REGIONAL"]
+  }
+}
+
+resource "aws_api_gateway_resource" "announcement_lambda" {
+  rest_api_id = aws_api_gateway_rest_api.management_api.id
+  parent_id   = aws_api_gateway_rest_api.management_api.root_resource_id
+  path_part   = "announcement_lambda"
+}
+
+resource "aws_api_gateway_method" "proxy" {
+  rest_api_id = aws_api_gateway_rest_api.management_api.id
+  resource_id = aws_api_gateway_resource.announcement_lambda.id
+  http_method = "POST"
+  authorization = "NONE"
+}
+
+resource "aws_api_gateway_integration" "lambda_integration" {
+  rest_api_id = aws_api_gateway_rest_api.management_api.id
+  resource_id = aws_api_gateway_resource.announcement_lambda.id
+  http_method = aws_api_gateway_method.proxy.http_method
+  integration_http_method = "POST"
```

Comment:
This is good for our use case, but a small FYI - its good practice to set the fixed values as variable with Default values other than writing it statically